core/src/test/kotlin/org/jetbrains/kotlinx/dataframe/rendering/RenderingTests.kt

```diff
@@ -196,4 +202,29 @@ class RenderingTests : TestBase() {
         val rendered = schema.toString()
         rendered shouldBe "a: Int?\nb: IntArray\nc: Array<Int>\nd: Array<Int?>"
     }
+
+    @Test
+    fun `render nested FormattedFrame as DataFrame`() {
+        val empty = object : CellAttributes {
+            override fun attributes(): List<Pair<String, String>> = emptyList()
+        }
+        val df = dataFrameOf("b")(1)
+
+        val formatted = dataFrameOf("a")(df.format { all() }.with { empty })
+        val nestedFrame = dataFrameOf("a")(df)
```

Comment:
It's probably okay, as formatting is usually the final step in the operation, but what would happen if you were to perform a format on a nested DF, then perform an operation on the frame column, like move, rename, or filter, and then convert it to html? Would it lose the formatting?

Reply:
Look at it this way: we had `api: DataColumn<DataFrame<Type>>`, and after this operation it becomes `api: DataColumn<FormattedFrame<Type>>` 
```
df.groupBy { receiverType }.toDataFrame("api")
    .convert { api }
    .with { it.format { all() }.where { refine || interpretable }.with { background(green) } }
```

Reply:
okay makes sense :)